src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -557,6 +564,7 @@ export class SavedObjectsService
         this.logger,
         migrationsRetryDelay
       ),
+      opensearchDashboardsRawConfig: this.opensearchDashboardsRawConfig as Config,
```

Comment:
I'd prefer to keep as it is because:
1. If we do some filter here, once we introduce more config to control the mappings dynamically, we need to change two places, filter and the `build_active_mapping`
2. For now the configService.atPath does not provide functionality to subscribe to multiple paths(even though it accepts string[], it is more like a lodash.get(obj, path)).